docs/index.md

```diff
@@ -1 +1,95 @@
+---
+layout: index
+---
 
+## Overview
+
+Marquez is an open source **metadata service** for the **collection**, **aggregation**, and **visualization** of a data ecosystem's metadata. It maintains the [provenance](https://en.wikipedia.org/wiki/Provenance#Data_provenance) of how datasets are consumed and produced, provides global visibility into job runtime and frequency of dataset access, centralization of dataset lifecycle management, and much more. Marquez<sup>[1](#1)</sup> was released and open sourced by [The We Company](https://www.we.co).
+
+## Why Marquez?
```

Comment:
Seems to me there needs to be a merger of this section and the tiny section below. The points in the below section seem to map to some of the points in this section. 

Maybe even going as far as making diagrams for this section too?

Reply:
Great feedback, @alexlingww!

> Seems to me there needs to be a merger of this section and the tiny section below. The points in the below section seem to map to some of the points in this section.

The `Features` section is actually a subsection of `Why Marquez?`. In the current layout, it might not be immediately obvious that the two sections are linked. I actually feel the `Features` section is a more appropriate subsection of `Overview`.

> Maybe even going as far as making diagrams for this section too?

Added a diagram illustrating a sample lineage query. More details can be added in future iterations.